xpdschema/schemas/run_start.json

```diff
@@ -0,0 +1,178 @@
+{
+  "type": "object",
+  "properties":{
+    "facility": {
+      "type": "string",
+      "description": "Facility Name (NSLS-II, APS, etc)"
```

Comment:
It is possible but it causes two issues a) we need to decide which nsls-ii name we go with and b) we need to name every facility that we are interested in working with. We might want/be able to put this into a separate document so we don't keep changing the schema itself just the table of 'valid' names.